MetaMask/metamask-extension
# UX: Display total fiat balance on home screen

## Explanation

Calculates and displays the total fiat value of a given account's native currency and tokens.

* Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1191


## Screenshots/Screencaps

<img width="679" alt="SCR-20230918-nvzn" src="https://github.com/MetaMask/metamask-extension/assets/46655/d766de4f-efca-4354-9aa1-f8a0f4d4b040">

<img width="667" alt="SCR-20230918-nyzh" src="https://github.com/MetaMask/metamask-extension/assets/46655/af1afa52-affe-4fc1-bd4b-a286097bf2b5">


## Manual Testing Steps

1.  Look at an account with native currency and token balances
2. See correct balance displayed above token list
3. Switch chains using the network picker
4. Ensure balance on L2 is correct

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
